be/src/bench/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
 # This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
 
-ADD_BE_BENCH(${SRC_DIR}/bench/chunks_sorter_bench)
-ADD_BE_BENCH(${SRC_DIR}/bench/runtime_filter_bench)
+#ADD_BE_BENCH(${SRC_DIR}/bench/chunks_sorter_bench)
```

Comment:
Why to annotate the two lines?

Reply:
ok, changed. comment can reduce compile time, i forgot to uncomment it when push commit.